vignettes/generation.Rmd

```diff
@@ -0,0 +1,170 @@
+
+Note that this method `coverageMethod = "PairedAndR1s"` option does use more RAM than using the `qsea` method. This can be set via `coverageMethod = "qseaPaired"`; single ended reads are not currently supported.
+
+### Note on Proper Pairs
+"Proper pairs", as determined by the `bwa mem` aligner at any rate, are paired-end sequencing reads where both ends have mapped sufficiently near to each other, and they point towards each other on opposite strands of the chromosome. However, it should be noted that how near is near enough to be called as proper pairs is calculated within `bwa mem` on an individual sample basis. This allowable distance is calculated by determining the mean and standard deviation of the fragment length amongst the mapped reads in that sample, with a cutoff of X standard deviations away from the mean. In general, for sheared genomic DNA this is not a big issue, because the distribution is sufficiently broad that the standard deviation is large enough. However, for circulating cell-free DNA, the distribution of fragment sizes has a very sharp peak at ~167bp, and hence the global fragment standard deviation is quite low. This can lead to read pairs of length ~400bp which otherwise look like they should be "proper pairs" not being called as such by the aligner. We therefore perform a rescuing process, to ensure that we keep these read pairs by finding read pairs that meet the orientation conditions for being proper pairs, but are within `minInsertSize` to `maxInsertSize` in length. This process is computationally expensive, so we only perform it when `properPairs = TRUE`.
+
+## Copy Number Variation
+
+The copy number variation is calculated over a set of larger genomic windows as set by `CNVwindowSize`; this is typically much larger than the `windowSize` due to coverage limitations. This will be performed using the same options as the enriched samples.
+Once the coverage is determined, the R package `hmmcopy` will be used, with default parameters, to calculate the copy number variation. The non-enriched (also known as "Input") bam files should be specified by adding a column called `input_file` to the `sampleTable`.
+
+Alternatively, `qsea` provides a method for determining the copy number from the methylation enriched samples, using only reads which do not contain a CG, this may be selected using `CNVmethod = "MeCap"`. We do note that for our MBD-seq experiments we found there was reasonable overall agreement, but this method did give spurious CNV compared to the non-enriched reads, although that may be due to the selectivity of our MBD-seq enrichment step being relatively high.
```

Comment:
`MeCap` is not mentioned as an available method in the help of the `makeQset` function.